src/ui/files_tab.rs

```diff
@@ -143,6 +145,13 @@ impl FilesTab {
         Ok(())
     }
 
+    pub fn untrack_file(&mut self, commander: &mut Commander) -> Result<()> {
+        self.file
+            .as_ref()
+            .map(|current_file| commander.untrack_file(&current_file));
```

Comment:
Let's handle an error from `commander.untrack_file`
```suggestion
            .map(|current_file| commander.untrack_file(&current_file))
            .transpose()?;
```